### DISCUSSION TITLE
Using the OS

### DISCUSSION THREAD

**Discussion by cbrown7752** [klange/toaruos#283] [General]
I downloaded the zip extracted it and put it onto a CD for use but I can't seem to figure out where to go next

**Comment by klange** [comment]
> downloaded the zip

ToaruOS is not distributed as a ZIP. What did you download?

> put it onto a CD

What did you put on the CD?

**Reply by cbrown7752** [reply]
Under releases it gives you an option for ISO tar.gz and a zip

**Reply by klange** [reply]
The tar.gz and ZIP options are automatically provided by Github and are clearly labeled as being source code. They are not the built operating system.

**Reply by cbrown7752** [reply]
What do I need to do then to get a working version on a CD?

**Reply by klange** [reply]
You need the ISO.

I do not recommend trying to make an actual CD of ToaruOS, as this is a waste of optical media which is becoming quite expensive due to its obsolescence. The ISO can be imaged to a USB storage device instead, possibly with tools like Rufus. Alternatively, ToaruOS's kernel and ramdisk can be extracted from the ISO and booted with an existing installation of Grub.

That said, I do not recommend running ToaruOS on real hardware due to the lack of device driver support. ToaruOS is intended to be run in virtual machines, and instructions for doing so with the ISO are listed in the README: https://github.com/klange/toaruos#running-toaruos
